src/app/components/source/UserSecurity.js

```diff
@@ -0,0 +1,522 @@
+                        }
+                      </span>
+                    </div>
+                  }
+                  {this.state.twoFactorAuthentication ?
+                    <span style={{ lineHeight: '25px' }}>
+                      {
+                        <FormattedMessage
+                          id="userSecurity.disableAuthenticateDescription"
+                          defaultMessage="Enter your password to disable two-factor authentication:"
+                        />
+                      }
+                    </span>
+                    : null }
+                  {this.state.showFactorCommonFields ?
+                    <TextField
+                      fullWidth
+                      type="password"
+                      name="password"
+                      required
+                      className="login__password-input"
+                      onChange={this.handleFieldChange.bind(this)}
+                      error={!this.state.errors.password}
+                      helperText={this.state.errors.password ? null : this.renderMessage('passwordError')}
+                      placeholder={this.renderMessage('passwordInput')}
+                    />
+                    : null
+                  }
+                  {this.state.twoFactorAuthentication ?
+                    <CardText style={cardTextAuthStyle}>
+                      <RaisedButton
+                        style={{ marginLeft: 'auto', marginRight: units(2) }}
+                        onClick={this.handleSubmitTwoFactorAuthentication.bind(this, false)}
+                        className="user-two-factor__enable-button"
+                        disabled={this.state.submitDisabled}
+                        label={
+                          <FormattedMessage id="userSecurity.disableTwofactor" defaultMessage="Disable" />
+                        }
+                      />
+                    </CardText>
+                    : null
+                  }
+                </CardText>
+                {this.state.showFactorAuthForm ?
+                  <div>
+                    <CardText style={cardTextAuthStyle}>
+                      <h3 style={subTitleStyle}>
+                        {
+                          <FormattedMessage
+                            id="userSecurity.downloadHeader"
+                            defaultMessage="Step 2: Download"
+                          />
+                        }
+                      </h3>
+                      <span style={{ lineHeight: '25px' }}>
+                        {
+                          <FormattedMessage
+                            id="userSecurity.downloadDescription"
+                            defaultMessage="You'll need a two-factor app, like Google Authenticator, on your smartphone to proceed:"
+                          />
+                        }
+                      </span>
+                      <Link to={appsUrls.apple} target="_blank" style={{ padding: '5px' }} >
```

Comment:
Since this is a external URL, I think it's better to use `<a href>` directly instead of `<Link />`... and please add the `rel="noopener noreferrer"` attribute to it

Reply:
Fixed